src/shortcuts/clearThought.js

```diff
@@ -4,6 +4,8 @@ import {
   setSelection,
 } from '../util'
 
+const tagsRegex = /<(?:.|\n)*?>/gmi
```

Comment:
We currently have two other regex definitions that strip tags/html. Could you assess whether they can be combined into a single regex, and if so, have them all (including `clearThought`) reference that regex from `constants`? Thanks!

- https://github.com/cybersemics/em/blob/dev/src/constants.ts#L194
- https://github.com/cybersemics/em/blob/dev/src/util/hashThought.ts#L8

Reply:
@raineorshine The `REGEXP_HTML` in constants `/<\/?[a-z][\s\S]*>/i` doesn't do strict tag checks. `<p>test>` as a whole will pass the regex test. Is this intentional ?

Reply:
`REGEXP_HTML` is currently used to detect HTML, not replace tags. I'm not sure what you mean by "strict tag checks."

You should be able to see exactly where each of the HTML-related regular expressions are being used, and determine if any of them can be combined based on their use.